bitnami/odoo/templates/pvc.yaml

```
@@ -8,8 +8,10 @@ metadata:
     {{- if .Values.commonLabels }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
+  annotations:
+  "helm.sh/resource-policy": {{ default "" .Values.persistence.resourcePolicy }}
```

I know that in the documentation says _(Quotation marks are required)_ but it works
```suggestion
    {{- if .Values.persistence.resourcePolicy }}
    helm.sh/resource-policy: {{ .Values.persistence.resourcePolicy | quote }}
    {{- end }}
```